src/helpers/date.ts

```diff
@@ -42,8 +42,10 @@ export function formatTime(time: string | number) {
 	return moment.utc(time).format(DateTimeMask.DATE_TIME_MASK);
 }
 
-export function timestampToNumber(timestamp: Timestamp): number {
-	return Math.floor(timestamp.epochSecond * 1000 + timestamp.nano / 1_000_000);
+export function timestampToNumber(timestamp: Timestamp | string): number {
```

Comment:
if this function was called with string typescript would have notified an error about it, so it is weird that we are changing this function param types without changing models or something else